Gemfile

```diff
@@ -1,12 +1,12 @@
 source 'https://rubygems.org'
 
-ruby '2.1.4'
+ruby '2.2.6'
```

Comment:
As per your commentary on the pull request, lets get this up to version 2.4 and merge?